app/queries/query_adapter/ephemera_vocabulary_query_adapter.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module QueryAdapter
+  class EphemeraVocabularyQueryAdapter < Base
+    def find_by(label:, vocabulary: nil)
```

Comment:
I think a generic `find_by` property which has multiple values you can pass it is setting a dangerous precedent. A special class for every model type seems like an early abstraction too which could lead to a lot of confusion about where to put some code and what interface is being expected at each layer. What's the import code look like without the class-specific `QueryAdapter`s? Is it also complex?